driver/src/test/java/org/neo4j/driver/ConfigTest.java

```diff
@@ -80,7 +80,7 @@ void shouldChangeToTrustedCert()
 
         // Then
         assertEquals( authConfig.strategy(), Config.TrustStrategy.Strategy.TRUST_CUSTOM_CA_SIGNED_CERTIFICATES );
-        assertEquals( trustedCert.getAbsolutePath(), authConfig.certFile().getAbsolutePath() );
+        assertEquals( trustedCert.getAbsolutePath(), authConfig.certFiles().get( 0 ).getAbsolutePath() );
```

Comment:
We aren't using AssertJ? That makes those assertions so much nicer.

Reply:
Not sure, probably just because it started this way a while ago.
However, we can probably consider introducing it in some areas.